apps/api-extractor/.heft/clean.json

```diff
@@ -1,11 +0,0 @@
-/**
```

Comment:
I see that we these deleted files got renamed to files like `api-extractor/config/clean.json`.  

Is that intentional? I though that this is now handled by  `"actionKind": "deleteFiles` from **heft.json**.

Reply:
Oops - I shouldn't have made this initial change yet. API Extractor has a cyclic dependency on Heft.